plugins/woocommerce/tests/php/includes/class-wc-mobile-messaging-handler-test.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/**
+ * Class WC_Mobile_Messaging_Handler_Test file.
+ *
+ * @package WooCommerce\Tests
+ */
+class WC_Mobile_Messaging_Handler_Test extends WC_Unit_Test_Case {
+
+	const BLOG_ID = 2;
+
+	/**
+	 * @var string $initial_country that is set on site which is a platform for unit tests, before running tests in this test suite.
+	 */
+	private static $initial_country = '';
+	/**
+	 * @var string $initial_currency that is set on site which is a platform for unit tests, before running tests in this test suite.
+	 */
+	private static $initial_currency = '';
+
+	/**
+	 * Saves values of initial country and currency before running test suite.
+	 */
+	public static function setUpBeforeClass(): void {
```

Comment:
```suggestion
	public static function wpSetUpBeforeClass(): void {
```

Probably best to use the specific WP child method for this instead of the generic phpunit one.